packages/angular_devkit/build_ng_packagr/src/build/index.ts

```diff
@@ -46,9 +47,15 @@ export class NgPackagrBuilder implements Builder<NgPackagrBuilderOptions> {
         getSystemPath(root), 'ng-packagr') as typeof ngPackagr;
       const packageJsonPath = getSystemPath(resolve(root, normalize(options.project)));
 
-      projectNgPackagr.ngPackagr()
-        .forProject(packageJsonPath)
-        .build()
+      const ngPkgProject = projectNgPackagr.ngPackagr()
+        .forProject(packageJsonPath);
+
+      if (options.tsConfig) {
```

Comment:
@filipesilva (about your first concern) except if I didn't tought about something, the tsConfig is not mandatory (you can see that the tests doesn't fail after the first commit), given that I first test if it's set before using it in the ngPackagr. Did I forgot some "optionnal" config somewhere ?